src/Zoomable.js

```diff
@@ -253,8 +253,8 @@ function getZoom(element) {
   let totalOffset = margin * 2;
   let viewport = getViewportDimensions();
 
-  let scaleX = viewport.width / (getWidth(element) + totalOffset);
-  let scaleY = viewport.height / (getHeight(element) + totalOffset);
+  let scaleX = Math.min(viewport.width / (getWidth(element) + totalOffset), element.dataset.actualWidth / getWidth(element));
+  let scaleY = Math.min(viewport.height / (getHeight(element) + totalOffset), element.dataset.actualHeight / getHeight(element));
```

Comment:
There is already a config available with `ignoreScroll`, so i think i should go there too.
but *actual* seems odd to me. I would prefer *original* or *natural* (devtools call them like that)